mobify/progressive-web-scaffold
# WEB-1112: Add CVV detection

**JIRA**:
https://mobify.atlassian.net/browse/WEB-1112

## Changes
- Add CVV detection

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to: [Payment step](https://www.merlinspotions.com/checkout/payment/)
- Toggle add a new card
- Try different credit card numbers like "4111 1111 1111 1111" or "3451 212391 23123" to see CVV change on the fly
